security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,62 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+[id="compliance-operator-1-7-0-bug-fixes_{context}"]
+=== Bug fixes
+
+* Before this release, Compliance Operator would provide an unneeded remediation recommendation due to differences in filesystem structure for the `s390x` architecture. With this release, the Compliance Operator now recognizes the differences in filesystem structure and does not provide the misleading remediation. With this update, the rule is now more clearly defined. (link:https://issues.redhat.com/browse/OCPBUGS-33194[*OCPBUGS-33194*])
+
+* Previously, the instructions for rule `ocp4-etcd-unique-ca` did not work for OCP 4.17 and later. With this update, the instructions and actionable steps are corrected. (link:https://issues.redhat.com/browse/OCPBUGS-42350[*OCPBUGS-42350*])
+
+* When using the Compliance Operator with Cluster Logging Operator (CLO) version 6.0, various rules would fail. This is due to backwards incompatible changes to the CRDs that CLO uses. The Compliance Operator relies on those CRDs to verify logging functionality. The CRDs have been corrected to support the PCI-DSS profiles with CLO. (link:https://issues.redhat.com/browse/OCPBUGS-43229[*OCPBUGS-43229*])
+
+* After installing logging 6.0, users found that the ComplianceCheckResult `ocp4-cis-audit-log-forwarding-enabled` was failing because there was a change in the APIversion of `clusterlogforwarder`. Log collection and forwarding configurations are now specified under the new API, part of the observability.openshift.io API group. (link:https://issues.redhat.com/browse/OCPBUGS-43585[*OCPBUGS-43585*])
+
+* For previous releases of Compliance Operator, the scans would generate an error log for the reconcile loop on the operator pod. With this release, the Compliance Operator controller logic is more stable. (link:https://issues.redhat.com/browse/OCPBUGS-51267[*OCPBUGS-51267*])
+
+* Previously, the rule `file-integrity-exists` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, the rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52884[*OCPBUGS-52884*])
+
+* Previously, the rule `file-integrity-notification-enabled` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, this rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52885[*OCPBUGS-52885*])
+
+* Before this release of the Compliance Operator, the rule `kubelet-configure-tls-cipher-suites` was failing in CI tests for the API server ciphers, resulting in `E2E-FAILURE` status. The rule was not recognizing new ciphers from RFC 8446. This has been resolved and the rule is now being evaluated correctly. (link:https://issues.redhat.com/browse/OCPBUGS-54212[*OCPBUGS-54212*])
```

Comment:
Similar comment here as above. We can consider removing the language around CI.

What I think readers might care about here is that the Compliance Operator has been updated so that `kubelet-configure-tls-cipher-suites` includes checking new ciphers from RFC 8446, that have been included as part of OpenShift 4.18. The broken behavior before was that the rule would fail because it thought the newly added ciphers we not secure, or safe, but they are and we just needed to update the rule accordingly.